test/unit-tests/components/structures/MatrixChat-test.tsx

```diff
@@ -134,10 +134,10 @@ describe("<MatrixChat />", () => {
             getVersion: jest.fn().mockReturnValue("1"),
             setDeviceIsolationMode: jest.fn(),
             userHasCrossSigningKeys: jest.fn(),
-            getActiveSessionBackupVersion: jest.fn().mockResolvedValue(null),
-            globalBlacklistUnverifiedDevices: false,
             // This needs to not finish immediately because we need to test the screen appears
             bootstrapCrossSigning: jest.fn().mockImplementation(() => bootstrapDeferred.promise),
+            getActiveSessionBackupVersion: jest.fn().mockResolvedValue(null),
+            globalBlacklistUnverifiedDevices: false,
```

Comment:
Why do we need to move theses two functions?

Reply:
Oh, we both added them and I probably accepted the wrong set of changes when merging.